### ISSUE TITLE
World is not saved if it is not unloaded

### ISSUE THREAD

**Issue by Kenuux** [InfernalSuite/AdvancedSlimePaper#6]
**Describe the issue**

The worlds are not saved unless you unload them. If one leaves the world and enters it again, all changes are gone. The command "/save-all" has no effect. Video: https://streamable.com/g3m51c

**Spigot version**

This server is running SlimeWorldManager version git-SlimeWorldManager-"edd8a4d" (MC: 1.19.3) (Implementing API version 1.19.3-R0.1-SNAPSHOT) (Git: edd8a4d)
Unknown version
Previous version: git-SlimeWorldManager-"9b65823" (MC: 1.19.3)

**Plugin version**
2.10.0

**How to reproduce**

- Load any world
- Do some changes 
- Leave server or join another world
- Get back to your world
- Changes were not made

**Crash reports (if available)**
/

**Comment by Kenuux** [comment]
Still not working

**Comment by Owen1212055** [comment]
> Still not working

Is this still the case?

**Comment by Kenuux** [comment]
> > Still not working
> 
> Is this still the case?
No, looks like it's fixed. Can't reproduce it since the related PR. 👍
